Messages/MessageTypes.h

```diff
@@ -15,5 +15,6 @@
 
 
 enum class MessageType {
```

Comment:
DataRequest = 0 Means that when you convert it to a integer that the enum values will start from 0 and count up. I tend to do it by habit